eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/CommonFundingHandlers.scala

```diff
@@ -40,17 +40,25 @@ trait CommonFundingHandlers extends CommonHandlers {
 
   this: Channel =>
 
-  def watchFundingSpent(commitment: Commitment, additionalKnownSpendingTxs: Set[ByteVector32] = Set.empty): Unit = {
+  def watchFundingSpent(commitment: Commitment, additionalKnownSpendingTxs: Set[ByteVector32], delay_opt: Option[FiniteDuration]): Unit = {
```

Comment:
```suggestion
  /**
   * @param delay_opt Optional delay to reduce herd effect at startup
   */
  def watchFundingSpent(commitment: Commitment, additionalKnownSpendingTxs: Set[ByteVector32], delay_opt: Option[FiniteDuration]): Unit = {
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/2733/commits/763720f8ac473768685438d3edf55c0d5b1bc371